packages/css-library/dist/tokens/css/variables.css

```diff
@@ -102,4 +102,57 @@
   --units-neg-9: -7.2rem;
   --units-neg-10: -8rem;
   --units-neg-15: -12rem;
+  --uswds-system-color-blue-5: #eff6fb;
+  --uswds-system-color-blue-10: #d9e8f6;
+  --uswds-system-color-blue-20: #aacdec;
+  --uswds-system-color-blue-30: #73b3e7;
+  --uswds-system-color-blue-40: #4f97d1;
+  --uswds-system-color-blue-50: #2378c3;
+  --uswds-system-color-blue-60: #2c608a;
+  --uswds-system-color-blue-70: #274863;
+  --uswds-system-color-blue-80: #1f303e;
+  --uswds-system-color-blue-90: #11181d;
+  --uswds-system-color-blue-vivid-50: #0076d6;
+  --uswds-system-color-blue-vivid-60: #005ea2;
+  --uswds-system-color-blue-cool-vivid-5: #e1f3f8;
+  --uswds-system-color-blue-cool-vivid-20: #97d4ea;
+  --uswds-system-color-blue-cool-vivid-40: #28a0cb;
+  --uswds-system-color-blue-cool-vivid-60: #07648d;
+  --uswds-system-color-blue-warm-vivid-60: #0050d8;
+  --uswds-system-color-blue-warm-vivid-70: #1a4480;
+  --uswds-system-color-blue-warm-vivid-80: #162e51;
+  --uswds-system-color-cyan-vivid-30: #00bde3;
+  --uswds-system-color-gold-vivid-5: #fef0c8;
+  --uswds-system-color-gray-1: #fcfcfc;
+  --uswds-system-color-gray-2: #f9f9f9;
+  --uswds-system-color-gray-3: #f6f6f6;
+  --uswds-system-color-gray-4: #f3f3f3;
+  --uswds-system-color-gray-5: #f0f0f0;
+  --uswds-system-color-gray-10: #e6e6e6;
+  --uswds-system-color-gray-20: #c9c9c9;
+  --uswds-system-color-gray-30: #adadad;
+  --uswds-system-color-gray-40: #919191;
+  --uswds-system-color-gray-50: #757575;
+  --uswds-system-color-gray-60: #5c5c5c;
+  --uswds-system-color-gray-70: #454545;
+  --uswds-system-color-gray-80: #2e2e2e;
+  --uswds-system-color-gray-90: #1b1b1b;
```

Comment:
This is a duplicate color with one that we had to manually add previously. Should be go ahead and just remove the duplicates while we're at it here? 

 `--v3-color-base-darkest: #1B1B1B;`

Reply:
There is also one other we manually added (it's that error red color `red-60v`):  

`--v3-color-error-dark: #b50909;`

https://github.com/department-of-veterans-affairs/component-library/blob/17280b3152a4b235fed72f6b610b24bf05a09e36/packages/css-library/dist/tokens/css/variables.css#L49C7-L50

Reply:
@Andrew565 Before merging, do you mind just linking to the follow-up ticket for this here? 🙏

Reply:
Follow-up ticket to replace color tokens with variables wherever possible: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2037